test/expected/ddl_errors.out

```diff
@@ -33,3 +33,32 @@ INSERT INTO "Hypertable_1" VALUES (0, 1, 0);
 ALTER TABLE _timescaledb_internal._hyper_1_1_chunk ALTER COLUMN temp_c DROP NOT NULL;
 ERROR:  operation not supported on chunk tables
 \set ON_ERROR_STOP 1
+CREATE UNLOGGED TABLE PUBLIC."Hypertable_unlogged" (
+  time BIGINT NOT NULL,
+  "Device_id" TEXT NOT NULL,
+  temp_c int NOT NULL DEFAULT -1
+);
+\set ON_ERROR_STOP 0
+SELECT * FROM create_hypertable('"public"."Hypertable_unlogged"', 'time', chunk_time_interval=>_timescaledb_internal.interval_to_usec('1 month'));
+ERROR:  table "Hypertable_unlogged" has to be logged
+\set ON_ERROR_STOP 1
+ALTER TABLE PUBLIC."Hypertable_unlogged" SET LOGGED;
+SELECT * FROM create_hypertable('"public"."Hypertable_unlogged"', 'time', chunk_time_interval=>_timescaledb_internal.interval_to_usec('1 month'));
+ create_hypertable 
+-------------------
+ 
+(1 row)
+
+CREATE TEMP TABLE "Hypertable_temp" (
+  time BIGINT NOT NULL,
+  "Device_id" TEXT NOT NULL,
+  temp_c int NOT NULL DEFAULT -1
+);
+\set ON_ERROR_STOP 0
+SELECT * FROM create_hypertable('"Hypertable_temp"', 'time', chunk_time_interval=>_timescaledb_internal.interval_to_usec('1 month'));
+ERROR:  table "Hypertable_temp" has to be logged
+ALTER TABLE "Hypertable_1" SET UNLOGGED;
+ERROR:  hypertables do not support unlogged tables
+ALTER TABLE "Hypertable_1" SET LOGGED;
+ERROR:  hypertables do not support unlogged tables
```

Comment:
Yeah, it seems counter-intuitive to generate the same error when doing `SET LOGGED`. Should be a noop, no?

Reply:
Fixed